apps/voting/app/src/components/MenuButton/MenuButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme } from '@aragon/ui'
+import IconMenu from './IconMenu'
+
+const StyledButton = styled.button`
+  border: none;
+  background: none;
+  margin-right: 8px;
+  height: 32px;
+  width: 32px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  cursor: pointer;
+  outline: none;
+
+  &:focus {
```

Comment:
Ideally we should use `:focus-visible` here, but it is not supported by Chrome yet, so we have [`<FocusVisible />`](https://github.com/aragon/aragon-ui/blob/master/src/components/FocusVisible/FocusVisible.js) that can be used in the meantime.

Usage:

```jsx
<FocusVisible>
  (({ focusVisible }) => (
    <div css={`
      :focus {
        border: `${Number(focusVisible)}px solid blue`;
      }
    `} />
  ))
</FocusVisible>
```

Reply:
Quick question: when Chrome starts supporting `:focus-visible` the `css` will work no?
What I am aiming for is that the solution would be to: a)keep the `css` rules, b) use the `FocusVisible` container and c) Remove `FocusVisible` when Chrome supports `:focus-visible`

Right? (what a mouth full, jajaja)

Reply:
The day Chrome starts supporting `:focus-visible`, using `FocusVisible` + `:focus` will still work, like it does in Firefox today. 👍

At this point, we will move away from `FocusVisible` + `:focus` and use `:focus-visible` instead.

e.g. in this particular case, we could have this today:

```jsx
 const StyledButton = styled.button`
  /* … */
  outline: 0;
  
  /* either wrap the entire :focus declaration: */
  ${p => p.focusVisible? `
    &:focus {
      outline: 2px solid ${theme.accent}`;
    }
  ` : ''}

  /* or on the value since there is only one: */
  &:focus {
    outline: ${p => p.focusVisible? `2px solid ${theme.accent}` : '0'};
  }
`

export default props => (
  <FocusVisible>
    (({ focusVisible }) => (
      <StyledButton focusVisible={focusVisible} {...props}>
        <IconMenu />
      </StyledButton>
    ))
  </FocusVisible>
)
```

And this tomorrow:


```jsx
 const StyledButton = styled.button`
  /* … */
  outline: 0;

  &:focus-visible {
    outline: 2px solid ${theme.accent};
  }
`

export default props => (
  <StyledButton focusVisible={focusVisible} {...props}>
    <IconMenu />
  </StyledButton>
)
```